x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group4/snooze.ts

```diff
@@ -367,28 +411,36 @@ export default function createSnoozeRuleTests({ getService }: FtrProviderContext
 
         const dateStart = new Date().toISOString();
         const snooze = {
-          ...SNOOZE_SCHEDULE,
-          rRule: {
-            ...SNOOZE_SCHEDULE.rRule,
-            // updating the dtstart to the current time because otherwise the snooze might be over already
-            dtstart: dateStart,
+          schedule: {
```

Comment:
This test fails as internal api used to create snooze schedule without id when there is no recurrence, and clear snooze schedules without ids when finished.
However now we always create a snooze schedule with ids regardless of recurrence. Hence it is never cleared. Should we proceed with this new behaviour or we should do same as internal api?

Reply:
removed this test